### DISCUSSION TITLE
Daily Statistics / Summary

### DISCUSSION THREAD

**Discussion by tiredofit** [stalwartlabs/smtp-server#35] [Q&A] [answered]
I am trying to match or exceed functionality from a long running postfix MTA, and run [pflogsumm](https://jimsun.linxnet.com/postfix_contrib.html) on a daily basis against my log for an easy to read summary report via cron like so:

`pflogsumm -d today --iso-date-time --problems-first -e --verbose_msg_detail --rej_add_from --zero_fill /logs//postfix.log | sudo -u postfix tee /logs//summary/$(TZ=America/Vancouver date '+%Y%m%d-%H%M%S')-mta_example_com.log`

Are there any plans or known script that would allow for parsing out daily statistics similar to above?

**Comment by stappersg** [comment]
> I am trying to match or exceed functionality from a long running MTA.

_“If I had asked people what they wanted, they would have said faster horses.”_     --  Henry Ford


And my actual message, my opinion:

It is all new and all different what is growing, please let the fresh idea grow by holding back questions based on a previous generation.


> Are there any plans or known script that would allow for parsing out daily statistics similar to above?

Scratch your itch.

**Comment by mdecimus** [comment, marked as answer]
> Are there any plans or known script that would allow for parsing out daily statistics similar to above?

Yes, once the webadmin is released  (Q1 2024) you will be able to collect statistics via an HTTP API.
